pip-wheel-metadata/libcst.dist-info/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I'm still confused on this file?

Reply:
I'm sorry, accidentally committed those. Just removed them.